cmd/cli/testdata/test-chart/templates/deployment.yaml

```diff
@@ -16,8 +16,8 @@ spec:
     spec:
       {{- with .Values.osm.imagePullSecrets }}
       nodeSelector:
-        kubernetes.io/arch: amd64
-        kubernetes.io/os: linux
+        kubernetes.io/arch: {{ .Values.osm.nodeSelector.arch }}
```

Comment:
FYI this test chart is only used for unit tests, so these changes aren't necessary.